torch/_functorch/aot_autograd.py

```diff
@@ -971,76 +1089,77 @@ def same_dtype_views(view1, view2):
-            # and error out. We can fix them later.
-            for idx1, idx2 in zip(aliased_input_indices, aliased_input_indices[1:]):
-                view1 = fwd_inputs[idx1]
-                view2 = fwd_inputs[idx2]
-                # The "inputs that are aliased but have different differentiable bases" case
-                # is more complicated and hopefully pretty rare. Not currently handled.
-                assert are_differentiable_views(view1, view2), \
-                    "aot_autograd() does not yet handle non-differentiable view input mutations."
-                # Regenerating views when reinterpreting complex / real tensors seems non-trivial,
-                # not handling for now
-                assert same_dtype_views(view1, view2), \
-                    "aot_autograd() does not yet handle input mutations on views with different dtypes."
-            non_none_bases = [fwd_inputs[i]._base for i in aliased_input_indices if fwd_inputs[i]._base is not None]
-            aliases_with_none_bases = [fwd_inputs[i] for i in aliased_input_indices if fwd_inputs[i]._base is None]
-            if len(non_none_bases) == 0:
-                # Case where none of the aliases require gradients
-                example_idx = aliased_input_indices[0]
-                synthetic_base = torch.Tensor(fwd_inputs[example_idx].storage())
-            else:
-                # Case where all of the aliases require gradients, and have the same _base.
-                synthetic_base = non_none_bases[0]
-                for other_base in non_none_bases[1:]:
-                    assert other_base is synthetic_base, \
-                        "aot_autograd() does not yet handle non-differentiable view input mutations."
-                for alias in aliases_with_none_bases:
-                    assert alias is synthetic_base, "aot_autograd() does not yet handle non-differentiable view input mutations."
-            base_args.append(synthetic_base)
-            for curr_view_idx in aliased_input_indices:
-                curr_view = fwd_inputs[curr_view_idx]
-                base_idx = len(base_args) - 1
-                size_ = curr_view.size()
-                stride_ = curr_view.stride()
-                storage_offset_ = curr_view.storage_offset()
-                # We store just enough info here so that we can regenerate the view later.
-                # Regeneration: args[base_idx].as_strided(size_, stride_, storage_offset_)
-                # If we want view replay instead of as_strided() calls, this will need to change.
-                inner_calling_convention_meta[curr_view_idx] = (base_idx, (size_, stride_, storage_offset_))
-        else:
             for curr_idx in aliased_input_indices:
                 other_args.append(fwd_inputs[curr_idx])
+            continue
+        # We detected an input that was mutated, AND aliases with another input.
```

Comment:
is there any change here besides reindenting?

Reply:
nope, just reindenting. I wish github difftool could figure that out 😛